MapboxNavigation/Style.swift

```diff
@@ -572,19 +572,19 @@ open class ManeuverContainerView: UIView {
     }
 }
 
+
 /// :nodoc:
-@objc(MBInstructionsBannerContentView)
-open class InstructionsBannerContentView: UIView { }
```

Comment:
Renaming these classes should be considered a backward incompatible change. Type alias or major bump? 🤔

Reply:
Major bump, I'm thinking.

Reply:
The classes aren’t documented, so they aren’t part of the public API, right? But this PR does add public, documented API, so we should bump the minor version anyways.